### DISCUSSION TITLE
Why @graphql-tools/utils is in dependencies instead of peerDependencies?

### DISCUSSION THREAD

**Discussion by valerii15298** [MichalLytek/typegraphql-nestjs#49] [General]
As I see in `type-graphql` package `@graphql-tools/utils` is in [devDependencies](https://github.com/MichalLytek/type-graphql/blob/master/package.json#L42). So why is it different in this project?
And does it make any impact(just curious about impact)?

**Comment by MichalLytek** [comment]
Because of this line:
https://github.com/MichalLytek/typegraphql-nestjs/blob/3fc003898343f6d7b710ef4c0a242ee45e19fd34/src/helpers.ts#L1
Which is required to make it work with apollo federation.
